recipe/meta.yaml

```diff
@@ -7,17 +7,17 @@ package:
 source:
     fn: armadillo-{{ version }}.tar.xz
     url: http://sourceforge.net/projects/arma/files/armadillo-{{ version }}.tar.xz
-    sha256: 10d1cc38503624bf1ad8c55c0d53b4feb80f04fcb3f5c3e6a69ba95d00f17966
+    sha256: 09962d8d22b98aea32633f6b6ad9b15189888b905f1f092183eda3c5bff35457
     patches:
         # have to add additional ${PREFIX}/lib etc... to tests/Makefile
         - make_tests.patch  # [linux]
         - make_tests_osx.patch  # [osx]
-        # modify CMakeLists to use OpenBLAS rather than Accelrate on OS X
-        # and to use OpenBLAS as the LAPACK library on linux
-        - CMakeLists.patch  # [unix]
+        # Temporary patch from upstream (remove when new version is released)
+        - macos_blas.patch
+        - macos_add_blas.patch
 
 build:
-  number: 1204
+  number: 0
```

Comment:
Could you please add a few things on ``requirements`` ``build``?
It would be better to add ``make`` for ``unix`` and ``nmake`` for ``windows``
```yaml
requirements:
  build:
    - make   # [unix]
    - nmake  # [win]
```

Reply:
As this is a new version, could you please double-check if it still needs to pin ``superlu``?
``superlu 5.2.* ``

Reply:
I check on their docs and it seems it is still necessary to pin ``superluc 5.2.*``